config/initializers/voyager_helpers.rb

```diff
@@ -1,5 +1,4 @@
 require 'yaml'
-require_relative '../../voyager_helpers/lib/voyager_helpers'
 include VoyagerHelpers::OracleConnection
 
 vh_config = YAML.load_file("#{File.dirname(__FILE__)}/../voyager_helpers.yml")
```

Comment:
This initializer is quite different from those instructions, though — parsing a YAML file instead of just doing it in ruby, assigning to vh_config instead of using VoyagerHelpers.configure.  Should we align marc_liberation with the config instructions?